main/guides/getting-started/swaparoo-how-to-pay-somone-explainer.md

```diff
@@ -0,0 +1,5 @@
+# Swaparoo Contract
+
+## How to Swap Assets
+Watch this short video to understand how payments are made between participants in the Swaparoo Smart Contract.
```

Comment:
ok, now the heading is on target, but the one-sentence summary doesn't line up well.

The walkthru covers quite a bit of stuff... the the whole invitation and offer handler flow, twice, with the second one required to match constraints from the first one; plus it includes "paying someone" in the form of sending them an invitation to act as the counter-party in the trade.

maybe...

```suggestion
Watch this short video walk-through of the complete Swaparoo Smart Contract that allows any two parties to trade any digital assets with minimal risk.
```

Reply:
Maybe the filename should change too?

Reply:
Did you consider renaming the file and decide against it?

Reply:
Oversight on my part, let me take care of that real quick...

Reply:
@dckc - filename changed